docs/dm/zh_CN/dm-worker-unit.md

```diff
@@ -0,0 +1,69 @@
+DM-Worker 详细介绍
+===
+
+### 简介
+- DM-Worker 可以链接到一台 MySQL/MariaDB，并且注册为该实例的 Slave
+- 支持读取 MySQL/MariaDB Binlog 持久化保存在本地（relay log）
+- 单个 DM-worker 支持同步一个上游 MySQL/MariaDB 到多个下游 TiDB
+- 多个 DM-Worker 支持同步多个 MySQL/MariaDB 到一个下游 TiDB 
+
+注意：如果需要合并多个上游 MySQL/MariaDB 的表到一张下游 TiDB 的表请参考 [分库分表](./shard-table))
+
+### DM-Worker 处理单元
+DM-Worker 任务运行过程包含多个任务处理逻辑单元
+
+#### relay log
+持久化保存从上游 MySQL/MariaDB 读取的 Binlog，并且对 binlog replication 处理单元提供读取 Binlog events 的功能
+
+原理和功能与 MySQL slave relay log 类似，参见 <https://dev.mysql.com/doc/refman/5.7/en/slave-logs-relaylog.html>。
+
+#### dumper
+从上游 MySQL/MariaDB dump 全量数据到本地磁盘
+
+#### loader
+读取 dumper 处理单元的数据文件，然后加载到下游 TiDB
+
+#### binlog replication/syncer
+读取 relay log 处理单元的 binlog events，转化为 SQLs，然后应用到下游 TiDB
+
+
+
+### DM-Worker 需要的权限
+包含 relay log、dump、load、replicate binlog 等处理单元， 这里先总体说下 上下游分别需要什么权限；
+
+#### 上游（MySQL/mariaDB)
+SELECT
+RELOAD
+RELICATION SLAVE
+REPLICATION CLIENT
```

Comment:
new white spaces or `-` needed to separate these lines.
you can view the rendered html to see the problem